front/components/assistant/Sharing.tsx

```diff
@@ -492,7 +492,7 @@ function SlackIntegrationDrawer({
         hasChanged={false}
         variant="side-sm"
       >
-        <div className="pt-8">
+        <div className="w-full max-w-4xl pt-8">
```

Comment:
🤔

Reply:
Can we add pt-8 in here:
https://github.com/dust-tt/dust/blob/dff04e365bd2f35fa9a22f7221de6ad679e567a2/front/components/sparkle/AppLayout.tsx#L192

Reply:
☝️ Meant for other places where `pt-8` makes sense.